src/content/packages-and-plugins/swift-package-manager/for-plugin-authors.md

```diff
@@ -0,0 +1,132 @@
+---
+title: Swift Package Manager for plugin authors
+description: How to add Swift Package Manager compatibility to iOS and macOS plugins
+diff2html: true
+---
+
+Flutter is migrating to Swift Package Manager to manage iOS and macOS native
```

Comment:
should we link to "Swift Package Manager" here, for moar better SEO and help those who aren't aware of it?

Reply:
Linked!